File: 4 - Estrutura de Repetição/Exercicio.20.py
#20. Fazer um programa que calcule e escreva o valor de S.

#      37*38      36*37     35*36           1*2
# S = -------- + ------- + ------- + ... + -----
#       1           2         3             37

S = 0.0
for i in range(1, 38):
    termo = (37 - i + 1) * (37 - i + 2) / i
    S += termo

print(f"O valor de S é: {S}")

File: Livro - Módulo Java/4 - Estrutura de Repetição/Exercicio20.java
public class Exercicio20 {
    public static void main(String[] args) {
        // Fazer um programa que calcule e escreva o valor de S.
        // S = 37*38/1 + 36*37/2 + 35*36/3 + ... + 1*2/37

        double S = 0.0;

        for (int i = 1; i < 38; i++) {
            double termo = (37 - i + 1.0) * (37 - i + 2.0) / i;
            S += termo;
        }

        System.out.println("O valor de S é: " + S);
    }
}